src/ui/src/data/recordQueue.ts

```diff
@@ -0,0 +1,37 @@
+import { Record as RLRecord } from "@/models/record";
+
+export async function getUnreviewedRecords(): Promise<RLRecord[]> {
+  // return (await fetch(`${API_URL}/demo/record`)).json();
+  // return [];
+  // return Promise.reject(new Error("oooops"));
+  return Promise.resolve([
+    {
+      id: "123",
+      patient: {
+        firstName: "John",
```

Comment:
Our API has been using snake case for property names, will NextJS automatically convert properties from snake case to camel?

Reply:
It will not, but please remember this ticket is not focusing on implementing the data structures as the APIs are defining it, nor it will take care of any API integration  or serialization/deserialization of data. All that will be worked on ticket #301 so unless is a blocking issue I would not worry about those items in this PR.